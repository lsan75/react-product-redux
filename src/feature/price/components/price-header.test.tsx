import * as Adapter from 'enzyme-adapter-react-16'
import * as React from 'react'

import { configure, shallow, ShallowWrapper } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import PriceHeader from './price-header'

describe('PriceHeader', () => {
  configure({ adapter: new Adapter() })
  let shallowOutput: ShallowWrapper

  beforeEach(() => {
    shallowOutput = shallow(
      <PriceHeader totalPrice={100} />
    )
  })

  afterEach(() => {
    shallowOutput.unmount()
  })

  it('should render', () => {
    expect(shallowToJson(shallowOutput)).toMatchSnapshot()
  })
})