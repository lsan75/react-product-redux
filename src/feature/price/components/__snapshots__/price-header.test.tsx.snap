// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PriceHeader should render 1`] = `
<article
  className="TotalPrice"
>
  <h2
    className="TotalPrice__Title"
  >
    Total
  </h2>
  <span
    className="TotalPrice__Value"
  >
    100
     â‚¬
  </span>
</article>
`;
